shell/linters/check-config.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+# Linters for stork.yaml and manifest.yaml
+CHECK_CONFIG=$("$DIR/gobin.sh" -p github.com/getoutreach/stork/cmd/check_config@v"$(get_tool_version "check_config")")
```

Comment:
nit: because of how this works it'd end up running on all yaml files, we should quickly change the linter format to also support `files=()`, which I can do if you don't have time after today.

Also, by setting `CHECK_CONFIG` here, when this is loaded to determine what extensions/(soon to be files) to run on, we'd end up compiling/downloading this tool. So, we probably want to put this into `check_cfg` or `linter` function instead.